code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -13,6 +13,18 @@
 	tastes = list("candy" = 1)
 	foodtype = JUNKFOOD | SUGAR
 
+/obj/item/reagent_containers/food/snacks/candy/bronx
+	name = "south bronx paradise"
+	desc = "Lose weight, guaranteed! Wait, the package actally says 'south bronx parasite'..."
```

Comment:
"actally"

should you capitalize references to the name of this candy bar? is it a brand or just a type of dish?

Reply:
Brand name, I... I guess? Should be fine now.